src/btree/bt_sync.c

```diff
@@ -68,15 +68,18 @@ __sync_checkpoint_can_skip(WT_SESSION_IMPL *session, WT_PAGE *page)
 static int
 __sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
 {
-	struct timespec end, start;
+	struct timespec end, start, cur_time;
 	WT_BTREE *btree;
 	WT_CONNECTION_IMPL *conn;
 	WT_DECL_RET;
 	WT_PAGE *page;
 	WT_REF *walk;
 	WT_TXN *txn;
+	uint64_t int_bytes_svpt, int_page_svpt, leaf_bytes_svpt, leaf_svpt;
```

Comment:
maybe variable names should be renamed. I am not getting a hint on what this variable is by "svpt".
leaf_svpt should be leaf_page_svpt to be consistent

Reply:
'svpt' is for savepoint, the value saved in a svpt gets used in the next log message when the stats has to be displayed in that message.

Reply:
I had no idea what `svpt` meant either.  My suggestion would be `_save`.

Reply:
I think I used this term because of my RDBMS background. It's used there. I will change it.